Updated the description for userIDs

brittnwestSAP opened the pull request:
Updated the description for userID and arrangerID in air and hotel based on behavior.
For air, the arrangerID = userID when traveler is booking for themselves. Otherwise it is the arrangerID.
For hotel, arrangerID will never be provided.

brittnwestSAP committed:
Updated the description for userIDs

Updated the description for userID and arrangerID in air and hotel based on behavior.
For air, the arrangerID = userID when traveler is booking for themselves. Otherwise it is the arrangerID.
For hotel, arrangerID will never be provided.

antonkushin commented on src/api-reference/common/event-pubsub/travel-search-event.markdown:
```diff
@@ -107,7 +107,8 @@ Name|Type|Format|Description
 ---|---|---|---
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In cases where the individual is booking on behalf of a guest, this is the user performing the search.
```
userId uniquely identifies the traveler. it is not necessarily the user who is performing the search. in a case of an arranger booking it would be the arrangerUserId who is actually performing the search.

brittnwestSAP replied:
looks good. Just added "e.g. arranger" to make it even more direct.

brittnwestSAP committed:
calling out user as arranger in hotel

Specifically calling out that this is the arranger - for easy reading.

charlieconcur commented on src/api-reference/common/event-pubsub/travel-search-event.markdown:
```diff
@@ -107,7 +107,8 @@ Name|Type|Format|Description
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In cases where the individual is booking on behalf of a guest, this is the user performing the search (e.g. the arranger).
+Unlike Air, Hotel will not return arranger information.
```
The hard line break doesn't work within a table structure. It needs to be in paragraph form and, on rare occasions, if you must, you can use `<br>` to make soft line breaks within the table.

brittnwestSAP replied:
Thanks! Fixed.

charlieconcur reviewed (changes requested):
A minor thing to fixup and then we will be ready.

brittnwestSAP committed:
adding line break

brittnwestSAP committed:
Updating formatting in userID notes

brittnwestSAP committed:
One more formatting update to userID

brittnwestSAP committed:
Updating wording for guest v arranger

If a guest, ID will be arranger ID
If an arranger, ID will be the traveler's ID

brittnwestSAP commented:
@antonkushin Can you confirm my last update was correct? I.e. If a guest, ID will be arranger ID
If an arranger, ID will be the traveler's ID

sarraloew commented on src/api-reference/common/event-pubsub/travel-search-event.markdown:
```diff
@@ -61,7 +61,7 @@ Name|Type|Format|Description
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
 `userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
-`arrangerUserId`|`String`|GUID|Uniquely identifies the user arranging the trip.
+`arrangerUserId`|`String`|GUID|If the user is also the traveler, this value will be the same as the userID above. If an arranger is booking on behalf of the traveler, this will uniquely identify the user arranging the trip. 
```
Change "userID" to "userId" to match the element capitalization.

sarraloew commented on src/api-reference/common/event-pubsub/travel-search-event.markdown:
```diff
@@ -107,7 +107,7 @@ Name|Type|Format|Description
 ---|---|---|---
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. <br> *NOTES:* </br>In cases where the individual is booking on behalf of a guest, a traveler without a Concur Travel profile, this is the user performing the search (e.g. the arranger).  In cases where the individual is booking on behalf of a traveler that does have a Concur profile (e.g. an arranger), this will be the arr ...
```
Update "on behalf of a guest, a traveler without a Concur Travel profile,"
to
"on behalf of a guest (a traveler without a Concur Travel profile),"
and update
"traveler that does have a Concur profile"
to
"traveler that does have a Concur Travel profile"

sarraloew reviewed (changes requested):
I've requested a few edits, let me know if you have any questions.

brittnwestSAP committed:
updated user ID description

Based on testing, it sounds like the traveler's ID is passed when booked by an arranger.

brittnwestSAP commented:
Resolved the comments. Ready for final review.
